src/chapters/enums/enums-in-models.rst

```diff
@@ -0,0 +1,191 @@
+   }
+
+Add an Enum Property to a Model Class
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Other objects can have enum type properties. To add an ``EventType`` property to our model ``Event``,
+we create a ``Type`` property in ``Event`` amongst the other properties declared:
+
+.. sourcecode:: csharp
+   :lineno-start: 12
+
+   // other Event property declarations
+
+   public EventType Type { get; set; }
+
+   // Event methods
+
+Once we have added an ``EventType`` property to our model, we need to add an ``EventType`` property to our ViewModel.
+
+Add an Enum Property to a ViewModel
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first thing we need to do is add an ``EventType`` property to ``AddEventViewModel``.
+
+.. sourcecode:: csharp
+   :lineno-start: 20
+
+   // other AddEventViewModel property declarations
+
+   public EventType Type { get; set; }
+
+Now that we have a property to hold the event type that the user selects, we need to create a list of all of the possible event types.
+We will use this list to populate the ``select`` element in a later step.
+
+.. sourcecode:: csharp
+   :lineno-start: 26
+
+   public List<SelectListItem> EventTypes { get; set; }
+
+   public AddEventViewModel()
+   {
+      EventTypes = new List<SelectListItem>();
+
+      EventTypes.Add(new SelectListItem
+      {
+            Value = EventType.CONFERENCE.ToString(),
+            Text = EventType.CONFERENCE.ToString()
+      });
+
+      EventTypes.Add(new SelectListItem
+      {
+            Value = EventType.MEETUP.ToString(),
+            Text = EventType.MEETUP.ToString()
+      });
+
+      EventTypes.Add(new SelectListItem
+      {
+            Value = EventType.SOCIAL.ToString(),
+            Text = EventType.SOCIAL.ToString()
+      });
+
+      EventTypes.Add(new SelectListItem
+      {
+            Value = EventType.WORKSHOP.ToString(),
+            Text = EventType.WORKSHOP.ToString()
+      });
+   }
+
+In the ``AddEventViewModel()`` constructor, we add each of the constants to the ``EventTypes`` list.
+``SelectListItem`` is a built-in class that can represent each item in the list in our ``select`` element.
+Each item in a ``select`` element uses the ``<option>`` tag.
+By setting ``Value`` in ``SelectListItem``, we are passing a value for the ``value`` attribute in the ``<option>`` tag.
+By setting ``Text`` in ``SelectListItem``, we are passing a value for the displayed text in the ``<option>`` tag.
+
+Pass Enum Values Through the Controller
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``CodingEvents`` uses model binding to create an ``AddEventViewModel`` object. So like any other field on 
+the model, the controller does not necessarily need to know about the addition of ``AddEventViewModel.Type`` in order to create an ``AddEventViewModel`` instance from a form.
+However, we need to make sure that we are properly setting the ``Type`` property of our ``Event`` object using the value from the ``Type`` property of our ``AddEventViewModel`` object.
```

Comment:
I suggest being explicit about which controller method we're talking about here.

Reply:
Done!